stl/inc/coroutine

```diff
@@ -10,12 +10,14 @@
 #if _STL_COMPILER_PREPROCESSOR
 
 #ifdef _RESUMABLE_FUNCTIONS_SUPPORTED
-#pragma message("The contents of <coroutine> are not available with /await.")
-#pragma message("Remove /await or use /await:strict for standard coroutines.")
-#pragma message("Use <experimental/coroutine> for legacy /await support.")
+#pragma message(_WARNING_MESSAGE("STL4038", "The contents of <coroutine> are not available with /await."))
+#pragma message(_WARNING_MESSAGE("STL4038", "Remove /await or use /await:strict for standard coroutines."))
+#pragma message(_WARNING_MESSAGE("STL4038", "Use <experimental/coroutine> for legacy /await support."))
 #else // ^^^ /await ^^^ / vvv no /await vvv
 #ifndef __cpp_lib_coroutine
-#pragma message("The contents of <coroutine> are available only with C++20 or later or /await:strict.")
+#define _MESSAGE "The contents of <coroutine> are available only with C++20 or later or /await:strict."
+#pragma message(_WARNING_MESSAGE("STL4038", _MESSAGE))
+#undef _MESSAGE // TRANSITION, DevCom-1479701

```

Comment:
Thanks for reporting this compiler bug! `_MESSAGE` is short enough that I worry about colliding with (evil :supervillain:) user code. I recommend something unique like `_STL4038_COROUTINE`.